index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
```

Comment:
Dear @Kgomotso196, kindly check the directory that you used for your style.css, you included the directory named "css" but in actual fact there's no folder named "css" in your project. You can create folder "css" and move your css file in that folder.

Other than that, I would suggest you remove that folder since you have not added it on your project. This is to help your html code link to the css file that has all the styles that need to be applied to your index.html. If you don't embed your css correctly its content won't be applied to your html file.

But you are almost there😊🥳